ui/apps/platform/src/Containers/VulnMgmt/Entity/TableWidgetFixableCves.js

```diff
@@ -30,7 +31,14 @@ const queryFieldNames = {
     [entityTypes.IMAGE_COMPONENT]: 'imageComponent',
 };
 
-const TableWidgetFixableCves = ({ workflowState, entityContext, entityType, name, id }) => {
+const TableWidgetFixableCves = ({
+    workflowState,
+    entityContext,
+    entityType,
+    name,
+    id,
+    vulnType = entityTypes.CVE,
```

Comment:
In the Deployments, Namespace, and Node sections, I think we would need to pass the correct vulnType, no?

Reply:
Correct. I deferred that--can be done in another PR.